AssetManagementTestProject/TestSetup/Constant.cs

```diff
@@ -15,8 +14,12 @@ public class Constant
     #region CREATED USERNAMES-PWS
     public const string STAFF_USERNAME = "anhp4";
     public const string STAFF_PASSWORD = "Staff@1234";
+    public const string CHANGED_USER_PASSWORD = "Staff@12345";
```

Comment:
These variables should have their own files in the Test Data folder e.g., LoginTestData.cs